m4_sessions/1710-inning/group_three.md

```diff
@@ -1,10 +1,28 @@
-Group Three
+## Group Three
 
-Session Title: 
-Description: Why should students come to this session? 
+### Session Title:
 
-Session Outcomes: What will students leave with or gain from this session?
+Global Decentralization (work-in-progress)
 
-Session Outline:  Include the flow of the session and the time for each section.
+### Description: Why should students come to this session?
 
-Names of group members 
+We feel that this is something that will be impacting more and more people as time goes on, and is a really untapped market for development opportunities. Also great for us to be aware of it as developers. 
+
+### Session Outcomes: What will students leave with or gain from this session?
+
+Students will leave with a better understanding of decentralization, how quickly and dramatically our world will change as a whole.  Students will have knowledge of different industry sectors.
+
+### Session Outline:  Include the flow of the session and the time for each section.
+
+Introduce the overarching topic, how to introduce this topic in the best way (?)
+Pick different products in different industries - have small group research sessions, and share outcomes with the rest of the class. Leaders spend time facilitate these small group discussions (15-20 minutes)
+Cover different sectors of the decentralization movement of our world. Large group discussion of pros and cons of outcomes from this movement. (10 minutes)
+Possible wrap-up at end of session, summary of everything we discussed in gear-up session (5-7 minutes).
```

Comment:
For sure do a whole group wrap up.  This will be important for students to hear from others.  Also might want to add a question of why is this important to you?  Or what more do you want to learn about decentralization?